data/json/faults.json

```diff
@@ -0,0 +1,78 @@
+[
+  {
+    "id": "fault_engine_belt_drive",
+    "type": "fault",
+    "name": "drive belt",
+    "description": "Required for operation of an attached alternator.",
+    "skills": [ [ "mechanics", 2 ] ],
+    "requirements": {
+      "qualities": [ { "id": "WRENCH", "level": 1 } ],
+      "components": [ [ [ "drivebelt", 1 ], [ "drivebelt_makeshift", 1 ] ] ]
+    }
+  },
+  {
+    "id": "fault_engine_glow_plug",
+    "type": "fault",
+    "name": "glow plugs",
+    "description": "Required to start the vehicle in low ambient tmperatures.",
```

Comment:
The description could be changed then. It suggests that it will fail to start completely rather than just waste time and power.